apps/staking/app/mystakes/modules/PriceModule.tsx

```diff
@@ -8,14 +8,22 @@ import {
   ModuleTitle,
 } from '@session/ui/components/Module';
 import { useTranslations } from 'next-intl';
+import { useWallet } from '@session/wallet/hooks/useWallet';
+import { arbitrumSepolia } from 'viem/chains';
+import ReferralModule from '@/app/mystakes/modules/ReferralModule';
 
 export default function PriceModule() {
+  const { chainId } = useWallet();
   const dictionary = useTranslations('modules.price');
   const generalDictionary = useTranslations('general');
   const titleFormat = useTranslations('modules.title');
 
   const title = dictionary('title');
 
+  if (chainId === arbitrumSepolia.id) {
+    return <ReferralModule />;
```

Comment:
We don't show the price if we are on testnet now?

Reply:
we wernt before, just moved the check to inside the price module instead of outside